src/components/AssignedMembers/hooks/useAssignedUsers/useAssignedUsers.js

```diff
@@ -0,0 +1,71 @@
+import { useQuery } from 'react-query';
+
+import {
+  useNamespace,
+  useOkapiKy,
+} from '@folio/stripes/core';
+
+import { MAX_RECORDS } from '../../../../constants';
+import { GROUPS_API, PERMISSIONS_API, USERS_API } from '../../constants';
+import { batchRequest, buildQueryByIds } from './utils';
+
+const useAssignedUsers = ({ grantedToIds = [], tenantId }, options = {}) => {
+  const ky = useOkapiKy();
+  const [namespace] = useNamespace({ key: 'get-assigned-users' });
+  const {
+    isLoading,
+    data = [],
+  } = useQuery(
+    [namespace, tenantId],
```

Comment:
What is the reason to use `tenantId` property only as a query key if your query always uses the current tenant id in the request headers? Probably you wanted to use `tenantId` as `X-Okapi-Tenant` header's value.

Reply:
I would suggest to put permissions set id instead of tenant it